text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/internal/parser/node/TagNode.java

```diff
@@ -68,7 +68,7 @@ public TagNode(
 
     // Then assert the tag node has a proper name.
     try {
-      TagInternals.sanitizeAndCheckTagName(this.name());
+      TagInternals.sanitizeAndAssertValidTagName(this.name());
```

Comment:
is this still needed? since we don't create tag nodes anyways for things with invalid names

Reply:
It never fails when running the minimessage test suite so let's assume it's not needed :p